contracts/vaults/BaseVaultWithSwap/RibbonThetaVaultWithSwap.sol

```diff
@@ -422,11 +455,19 @@ contract RibbonThetaVaultWithSwap is RibbonVault, RibbonThetaVaultStorage {
 
         emit OpenShort(newOption, lockedBalance, msg.sender);
 
-        VaultLifecycleWithSwap.createShort(
-            GAMMA_CONTROLLER,
-            MARGIN_POOL,
+        uint256 optionsMintAmount =
+            VaultLifecycleWithSwap.createShort(
+                GAMMA_CONTROLLER,
+                MARGIN_POOL,
+                newOption,
+                lockedBalance
+            );
+
+        VaultLifecycleWithSwap.allocateOptions(
```

Comment:
To reduce the complexity this time around, can we comment out code for `allocateOptions`?

Reply:
We can add it back in later when we launch the purchase queue